src/fs.ts

```diff
@@ -9,12 +9,12 @@ const writeFileAsync = util.promisify(fs.writeFile);
 const existsAsync = util.promisify(fs.exists);
 
 export async function exists(path: string, cwd = ''): Promise<boolean> {
-  return existsAsync(join(cwd, path));
+  return await existsAsync(join(cwd, path));
 }
 
 export async function readText(path: string, cwd = ''): Promise<string> {
   try {
-    return readFileAsync(join(cwd, path), { encoding: 'utf8' });
+    return await readFileAsync(join(cwd, path), { encoding: 'utf8' });
```

Comment:
I think that await is needed on line 17 because without it catch will never be called while on line 12 we don't have to handle the async operation but we simply return it

Reply:
Correct, it's not required on line 12, but I left it there for consistency with the others and one less way for it to go wrong when refactoring. The same is true for line 71.

If you really insist, I will change it, otherwise I'm happy with it as-is.

Reply:
is good for me, it was a suggestion if you didn't know it 😄